rest-api-spec/src/main/resources/rest-api-spec/api/search.json

```diff
@@ -154,6 +154,10 @@
         "request_cache": {
           "type" : "boolean",
           "description" : "Specify if request cache should be used for this request or not, defaults to index level setting"
+        },
+        "search_after": {
+          "type" : "list",
+          "description" : "A comma-separated list of sort values that indicates where the sort of the top hits should start"
```

Comment:
@jimferenczi the default value returned in the REST API for a missing string value is `null`